src/main/java/Main.java

```diff
@@ -0,0 +1,6 @@
+public class Main {
```

Comment:
Typically Java projects don't use classes without package. A lot of times the packages correspond to domains the person/company owns. What about `me.karakelley.tictactoe`.